CIP-XXXX/README.md

```diff
@@ -0,0 +1,312 @@
+---
+CIP: <?>  
+Title: <Oracle Datum Standard>  
+Authors: Cardano DeFi Alliance <info@cardanodefialliance.org>, Cody Butz <cbutz@indigo-labs.io>, Jonas Lindgren <jonas@charli3.io>
+Comments-Summary: <No comments>  
+Comments-URI: <>  
+Status: <Draft>  
+Type: <Standards>  
```

Comment:
```suggestion
Type: Process  
```

Reply:
I'm confused by the suggestion to change from Standards to Process, isn't this very clearly in the Standards camp? Am I misunderstanding the definition of Standards vs Process, or have I missed context?

Reply:
Will do, thank you.

Reply:
Thanks for edits... going back now & resolving conversations relating to preamble format.